bl-service/src/main/java/com/ayeeti/blservice/user/UserController.java

```diff
@@ -34,4 +35,14 @@ public UserDTO updateUser(@RequestBody UpdateUserRequest userUpdateUserNameReque
         return userService.updateUser(userUpdateUserNameRequest.userId(), userUpdateUserNameRequest.newUserName());
     }
 
+    @PostMapping("/createUserAtLocation")
```

Comment:
Again you are not naming your endpoints mapping correctly. We've been over this and I explained it 2 times already.